playbooks/roles/jenkins/templates/aws_config.j2

```diff
@@ -0,0 +1,7 @@
+{% for role in assume_roles %}
```

Comment:
Not sure you're going to be able to assume this role without adding jenkins here:
https://github.com/alphagov/digitalmarketplace-aws/blob/master/terraform/modules/cloudtrail/modules/cloudtrail-validate-logs-role/main.tf#L4-L3

Reply:
I don't massively dislike the paradigm but we could just add the policy statements here
https://github.com/alphagov/digitalmarketplace-aws/blob/master/terraform/modules/cloudtrail/modules/cloudtrail-validate-logs-role/main.tf#L19
 to the jenkins instance profile here:
https://github.com/alphagov/digitalmarketplace-aws/blob/master/terraform/accounts/main/jenkins_instance_profile.tf#L39

Reply:
_I think_

Reply:
> I don't massively dislike the paradigm but we could just add the policy statements here
> https://github.com/alphagov/digitalmarketplace-aws/blob/master/terraform/modules/cloudtrail/modules/cloudtrail-validate-logs-role/main.tf#L19
> to the jenkins instance profile here:
> https://github.com/alphagov/digitalmarketplace-aws/blob/master/terraform/accounts/main/jenkins_instance_profile.tf#L39

The reasoning was that it makes it easier for anyone to run the task, not just the Jenkins box

Reply:
> Not sure you're going to be able to assume this role without adding jenkins here:
> https://github.com/alphagov/digitalmarketplace-aws/blob/master/terraform/modules/cloudtrail/modules/cloudtrail-validate-logs-role/main.tf#L4-L3

Oh yeah haven't got around to that yet 😆